duckduckgo/Android
# User Segment calculations

Task/Issue URL: https://app.asana.com/0/72649045549333/1206955287217878/f

### Description
Add user segmentation

### Steps to test this PR
The tests added should ensure the functionality works as expected.

To run manual test, one needs to set up charles and proxy/change the `/atb.js` response
